utils/sentry.py

```diff
@@ -17,7 +21,7 @@ def log_message(message: str, details: str | dict[str, Any] | None = None, level
             capture_message(message, level=level)
 
     @staticmethod
-    def log_exception(err: Exception, details: str, **extra) -> None:
+    def log_exception(err: Exception, details: str, **extra: dict[str, Any]) -> None:
```

Comment:
`**kwargs` should be typed by their value, so `**extra: Any` for keyword arguments of `Any` type.

Reply:
Oopsie, nice catch 😄